internal/tiltfile/value/string.go

```diff
@@ -17,31 +21,31 @@ func AsString(x starlark.Value) (string, bool) {
 
 // Unpack an argument that can either be expressed as
 // a string or as a list of strings.
-func AsStringOrStringList(x starlark.Value) ([]string, bool) {
+func AsStringOrStringList(x starlark.Value) ([]string, error) {
 	if x == nil {
-		return []string{}, true
+		return []string{}, nil
 	}
 
 	s, ok := AsString(x)
 	if ok {
-		return []string{s}, true
+		return []string{s}, nil
 	}
 
-	iterable, ok := x.(starlark.Iterable)
-	if ok {
-		result := []string{}
-		iter := iterable.Iterate()
-		defer iter.Done()
-		var item starlark.Value
-		for iter.Next(&item) {
-			s, ok := AsString(item)
-			if !ok {
-				return nil, false
-			}
-			result = append(result, s)
-		}
-		return result, true
+	list, ok := x.(*starlark.List)
```

Comment:
is it also important to handle tuples or nah?

Reply:
I could go either way. I'm not sure whether allowing them or not is more pythonic (or starlarkic?). Python allows `for` over tuples, so I've added them (though it also allows `for` over dicts, so that's not necessarily much of an argument - at least they won't fail weirdly in this case like dicts do)